Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -225,7 +230,7 @@ public override bool ConnectALMServer()
                 {
                     if (string.IsNullOrEmpty(item.SelectedValue)|| item.SelectedValue=="Unassigned")
                     {
-                        item.SelectedValue= defectForOpening.Value.ContainsKey(item.ExternalID) && defectForOpening.Value[item.ExternalID]!= "Unassigned" ? defectForOpening.Value[item.ExternalID] : string.Empty;
+                        //item.SelectedValue= defectForOpening.Value.ContainsKey(item.ExternalID) && defectForOpening.Value[item.ExternalID]!= "Unassigned" ? defectForOpening.Value[item.ExternalID] : string.Empty;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6361231)